tests/samplers_tests/tpe_tests/test_multi_objective_sampler.py

```diff
@@ -515,7 +515,7 @@ def test_calculate_weights_below_for_multi_objective() -> None:
     assert np.isclose(weights_below[1], weights_below[2])
     assert sum(weights_below) > 0
 
-    # +/-inf objective values.
+    # Test -inf objective values.
```

Comment:
The test includes `+inf`. Should we remove `[float("inf"), float("inf")]` to make it consistent with the comment?

Reply:
Sorry, I'll update the comments.
```suggestion
    # Test +/-inf objective values.
```